apps/client/components/userStats/UserStats.tsx

```diff
@@ -0,0 +1,51 @@
+'use client'
+
+import { useState } from 'react'
+
+import { Announcement } from '../announcements/Announcement'
+import { PrimaryButton } from '../ui/PrimaryButton'
+
+export const UserStats = () => {
+  const [isSelected, setIsSelected] = useState('announcements')
```

Comment:
nazwa sugeruje ze to boolean a jednak to string. Po za tym typowanie jest nie poprawne, nie wiem jak nie masz niżej błędu